libdistopia/src/distopia.cpp

```diff
@@ -460,24 +451,27 @@ namespace distopia {
                     n2x, n2y, n2z,
                     xp_x, xp_y, xp_z);
 
+
             auto x = hn::Zero(d);
-            hn::MulAdd(n1x, n2x, x);
-            hn::MulAdd(n1y, n2y, x);
-            hn::MulAdd(n1z, n2z, x);
+            x = hn::MulAdd(n1x, n2x, x);
+            x = hn::MulAdd(n1y, n2y, x);
+            x = hn::MulAdd(n1z, n2z, x);
 
             auto vb_norm = hn::Zero(d);
-            hn::MulAdd(bx, bx, vb_norm);
-            hn::MulAdd(by, by, vb_norm);
-            hn::MulAdd(bz, bz, vb_norm);
+            vb_norm = hn::MulAdd(rbc_x, rbc_x, vb_norm);
+            vb_norm = hn::MulAdd(rbc_y, rbc_y, vb_norm);
+            vb_norm = hn::MulAdd(rbc_z, rbc_z, vb_norm);
             vb_norm = hn::Sqrt(vb_norm);
 
             auto y = hn::Zero(d);
-            hn::MulAdd(xp_x, bx, y);
-            hn::MulAdd(xp_y, by, y);
-            hn::MulAdd(xp_z, bz, y);
+            y = hn::MulAdd(xp_x, rbc_x, y);
+            y = hn::MulAdd(xp_y, rbc_y, y);
+            y = hn::MulAdd(xp_z, rbc_z, y);
+            
             y = y / vb_norm;
 
-            return hn::Atan2(d, y, x);
+            // negate due to vector order (?)
```

Comment:
We need to take a negative here, which I think stems from the vector order used here, we use r_ab whereas `calc_distances` uses `-r_ab`

Reply:
hmm, this is good but I might see if we can rearrange above to not require the `hn::Neg`` here